client/QuickStart/API_Util.cs

```diff
@@ -0,0 +1,410 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Security.Cryptography.X509Certificates;
+using Kusto.Data;
+using Kusto.Data.Common;
+using Kusto.Ingest;
+using Newtonsoft.Json;
+using Kusto.Cloud.Platform.Data;
+using ShellProgressBar;
+
+namespace QuickStart
+{
+    /// <summary>
+    /// ConfigJson object - represents a cluster and DataBase connection configuration file.
+    /// </summary>
+    public class ConfigJson
```

Comment:
This class and it's read method should be part of the app itself (not a reusable kusto code)

Reply:
Maybe move this to a third file?